checkout-app/src/styles.scss

```diff
@@ -22,45 +63,35 @@ button {
   }
 }
 
-.btn-primary,
-.btn-primary:disabled {
-  background-color: #553ed8;
-  border-color: #553ed8;
-  &:hover,
-  &:focus,
-  &:active {
-    background-color: #2913a5 !important;
-    border-color: #2913a5 !important;
-  }
+.btn {
+  font-weight: bold;
+  padding: 10px;
+  width: 200px;
```

Comment:
Can we get rid of this fixed width?
Should avoid it in general for buttons as its not responsive-friendly and can cause downstream impacts, for example: 
![image](https://user-images.githubusercontent.com/24974768/82936436-7ffde080-9f86-11ea-8879-dd44099d60bc.png)

We should let the button scale based on content and trust bootstrap to handle sizing. Only override if necessary :)

Reply:
Sorted.